### ISSUE TITLE
Bug when parsing already linked urls with ?

### ISSUE THREAD

**Issue by xdamman** [gregjacobs/Autolinker.js#263]
```js
var autolinker = require('autolinker');

let html = autolinker.link(`<a href=https://www.facebook.com/hashtag/muntcentrum>#Muntcentrum</a>`);
console.log(html); // works: "<a href=https://www.facebook.com/hashtag/muntcentrum>#Muntcentrum</a>"

html = autolinker.link(`<a href=https://www.facebook.com/hashtag/muntcentrum?epa=HASHTAG>#Muntcentrum</a>`);
console.log(html); // fails: "<a href=<a href=\"https://www.facebook.com/hashtag/muntcen….com/hashtag/muntcentrum?epa=HASHTAG</a>>#Muntcentrum</a>"

```

See: https://runkit.com/xdamman/5c5a51eee4b0c90013568f62

**Comment by gregjacobs** [comment]
Wow, that was an oversight :X  Looks like a problem with the html parsing of unquoted attributes. Will look into it as soon as I get a chance

**Comment by gregjacobs** [comment]
Hey @xdamman, this is fixed in v3.0.1. Interestingly, it was the `=` character that was breaking it rather than the `?` character :)

**Comment by xdamman** [comment]
Thank you! 🙏 

(Since I couldn’t find a place to donate, I just made a pledge on https://opencollective.com/autolinker )
